### ISSUE TITLE
pylint(no-member) when *_pb2.py is in a package

### ISSUE THREAD

**Issue by pw1316** [nelfin/pylint-protobuf#19]
Here is my project structure:

./pkg/test.proto
```
message Person {
  required string name = 1;
  required int32 id = 2;
  optional string email = 3;
}
```

./pkg/test_pb2.py (generated)

./main.py
```
import pkg
import pkg.test_pb2 as test_pb2
from pkg.test_pb2 import Person

person = test_pb2.Person(name='a')
print(person.name)  # fine
person = pkg.test_pb2.Person(name='b')
print(person.name)  # pylint(no-member)
person = Person(name='b')
print(person.name)  # fine
```

Why the second one gives the warning and can it be fixed?

**Comment by nelfin** [comment]
Hi @pw1316, thanks for the report. `no-member` is not raised by `pylint-protobuf`, it is built-in. Normally we try to suppress it when we know that it shouldn't apply, see #2. This case seems to be caused by same underlying issue affecting #16 and others: namespaces aren't really tracked very well internally (which is why when you re-import the module it works, but not when you refer to it as an attribute of the package). I've been working on fixing this over the last two weeks, but I haven't had as much time lately.

**Comment by nelfin** [comment]
This appears to be fixed in v0.14 and later
